objectbox/lib/src/modelinfo/modelproperty.dart

```diff
@@ -50,8 +50,25 @@ class ModelProperty {
 
   set dartFieldType(String value) => _dartFieldType = value;
 
-  String get fieldType =>
-      _dartFieldType!.replaceFirst('?', '', _dartFieldType!.length - 1);
+  // Used to inform about an ID field that is read-only (needed for code-gen).
+  // We're adding this prefix to [_dartFieldType] so that we don't have to do
```

Comment:
Update `_dartFieldType` docs to indicate this (also doesn't mention the `?`, yet)?

Reply:
updated the docs.